src/Quadrature/quadrature.jl

```diff
@@ -16,8 +16,15 @@ using Base.Cartesian: @nloops, @ntuple, @nexprs
 
 Create a `QuadratureRule` used for integration on the refshape `shape` (of type [`AbstractRefShape`](@ref)).
 `order` is the order of the quadrature rule.
-`quad_rule_type` is an optional argument determining the type of quadrature rule.
-Currently the :polyquad, `:legendre` and `:lobatto` rules are implemented, depending on the reference element.
+`quad_rule_type` is an optional argument determining the type of quadrature rule,
+currently the `:legendre` and `:lobatto` rules are implemented for hypercubes.
+For triangles up to order 8 the default rule is the one by `:dunavant` (see [Dun:1985:hde](@cite)) and for
+tetrahedra the default rule is `keast_positive` (see [Keast:1986:mtq](@cite)). Wedges and pyramids default
+to `:polyquad` (see [WitVin:2015:isq](@cite)).
+Furthermore we have implemented
+* `:gaussjacobi` for triangles (order 9-15)
+* `:jinyun` (see [Jin:1984:sgq](@cite)) for tetrahedra (order 1-4)
+* `:keast_minimal` (see [Keast:1986:mtq](@cite)) for tetrahedra (order 1-5)
```

Comment:
The references don't work well in the docstrings, so I'd suggest only mentioning the defaults here, and refer the user to the online docs for further details. 
There we could have a table or similar showing the defaults and options for each reference shape, as well as the literature references and possibly some key properties of the rules.

Reply:
Let us leave this as a separate PR because this is a bigger change. Especially working out the properties.